### ISSUE TITLE
Camera feed

### ISSUE THREAD

**Issue by dascar5** [AlexeyAB/darknet#2032]
I got it to work on already downloaded files, but I just can't seem to use it through smartwebcam
I installed it and checked that it worked correctly through my browser, but when I do 
`darknet_no_gpu.exe detector demo data/coco.data yolo-voc.cfg yolo-voc.weights http://192.168.1.6:8080/video?dummy=param.mjpg -i 0`
I get some preloading just like when I do it with a picture, but instead of showing me results, the cmd just closes.
Any tips?

**Comment by AlexeyAB** [comment]
Find files opencv_world320.dll and opencv_ffmpeg320_64.dll (or opencv_world340.dll and opencv_ffmpeg340_64.dll) in C:\opencv_3.0\opencv\build\x64\vc14\bin and put it near with yolo_console_dll.exe

----


Add to cmd file this content:
```
darknet_no_gpu.exe detector demo data/coco.data yolo-voc.cfg yolo-voc.weights http://192.168.1.6:8080/video?dummy=param.mjpg -i 0

pause
```

then run it, make screenshot and attach it to your message.

**Comment by dascar5** [comment]
Ok, so, 2 things. 
I don't have the yolo_console_dll.exe
I saw the sln file though, tried to build it, and it told me `Cannot open include file: 'opencv2/opencv.hpp': No such file or directory
`
Which is weird since I never got opencv errors in any of the steps so far.

Skipped that step, tried your pause thing next, and this is what I got
https://imgur.com/CQYPCbG

**Comment by AlexeyAB** [comment]
As described here you should use `voc.data` file instead of `coco.data` file for `yolo-voc.cfg` model: https://github.com/AlexeyAB/darknet#for-using-network-video-camera-mjpeg-stream-with-any-android-smartphone

Use this command
`darknet_no_gpu.exe detector demo data/voc.data yolo-voc.cfg yolo-voc.weights http://192.168.1.6:8080/video?dummy=param.mjpg -i 0`

**Comment by dascar5** [comment]
Works perfectly lol, except it eats 100% of my cpu and lags like hell, but I'm guessing there's nothing that can be done with that?
Anyway, great job, and thanks!

**Comment by AlexeyAB** [comment]
@dascar5 Buy nVidia GPU and compile `darknet.sln` to solve this issue.

**Comment by dascar5** [comment]
Yeah I already have a GTX1080 on my PC but I need to present this on my
laptop 😁

**Comment by AlexeyAB** [comment]
Download this weights: https://pjreddie.com/media/files/yolov3-tiny.weights

And use this command
`darknet_no_gpu.exe detector demo cfg/coco.data yolov3-tiny.cfg yolov3-tiny.weights http://192.168.1.6:8080/video?dummy=param.mjpg -i 0`

**Comment by dascar5** [comment]
I really appreciate how you actually help people unlike most users who
don't even bother responding. Wish you the best dude.

**Comment by dascar5** [comment]
Regarding that, would it be possible to cut out all the extra features?
Like, I just need it to count cars, anything else is overkill.
